seqr/views/apis/saved_variant_api_tests.py

```diff
@@ -160,9 +160,9 @@ def test_create_saved_variant(self):
         create_saved_variant_url = reverse(create_saved_variant_handler)
         self.check_collaborator_login(create_saved_variant_url, request_data={'familyGuid': 'F000001_1'})
 
-        variant_json = {
+        core_variant_json = {
```

Comment:
Why don't you make it a constant and put it at the beginning of the file?

Reply:
because I was trying to make as few changes as possible and this was already here. But it would be better if it was a constant, so I can make that change